tests/data/group_system/group_network/group_network-kernel/group_network_host_and_router_parameters/rule_sysctl_net_ipv4_conf_default_secure_redirects/line_not_there.fail.sh

```diff
@@ -3,3 +3,5 @@
 # profiles = xccdf_org.ssgproject.content_profile_C2S, xccdf_org.ssgproject.content_profile_cui
 
 sed -i "/^net.ipv4.conf.default.secure_redirects.*/d" /etc/sysctl.conf
+
```

Comment:
The profiles in this scenario haven't been changed. Is that intentional? I can see `sysctl_net_ipv4_conf_default_secure_redirects` is RHEL7 OSPP.

Reply:
I probably forgot to change. I'll do it.